dataUtil/featureExtraction.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+from scipy.signal import spectrogram as spectro
+from dataUtil.processing import read_audio_data
+from librosa.feature import mfcc
+from librosa.display import specshow
+import matplotlib.pyplot as plt
+
+
+def get_fft(signal, sampling_frequency):
+    '''
```

Comment:
PEP docstring conventions say to use “ over ‘. Let’s be consistent across files.
https://www.python.org/dev/peps/pep-0257/